test/test_helper.exs

```diff
@@ -1,3 +1,24 @@
-Application.ensure_all_started(:ex_machina)
-ExUnit.start(exclude: [:skip])
-Ecto.Adapters.SQL.Sandbox.mode(Oli.Repo, :manual)
+defmodule Oli.TestHelper do
+  def configure_tests(:hound) do
+    File.mkdir_p!("test-results/screenshots")
+    Application.ensure_all_started(:ex_machina)
+    Application.ensure_all_started(:hound)
+    ExUnit.start(exclude: [:test], include: [:hound])
+
+    # We can't use manual SQL sandbox mode here, because torus attempts to run
+    # several queries at startup that fail in that mode. However, the tests do
+    # requre the SQL sandbox, so we set up a shared strategy here at the outset.
+    # This means we should refrain from making these tests async for now.
+    # TODO: What's a better way to handle this?
```

Comment:
Reading a little more the [sandbox docs](https://hexdocs.pm/ecto_sql/Ecto.Adapters.SQL.Sandbox.html#module-collaborating-processes), if we want to make possible to run tests async, we should leave the `manual` mode and `allow` the processes trying to connecto to the db. It might be a bit tricky to do this, so if you need any help let me know.

Reply:
Quick note for follow-up: https://github.com/phoenixframework/phoenix_ecto#concurrent-browser-tests